assets/prototype/domain/eventEditor/datetimes/dateList/filter/DatesListEntityFilters.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Internal imports
+ */
+import DatesSortedByControl from './controls/DatesSortedByControl';
+import DisplayDatesControl from './controls/DisplayDatesControl';
+import ShowDatesControl from './controls/ShowDatesControl';
+
+/**
+ * filters for controlling the display of a list of Event Dates
+ *
+ * @param {string} showDates
+ * @param {string} datesSortedBy
+ * @param {string} displayDates
+ * @return {Object} EditorDatesListView with added DateListFilters
+ */
+const DatesListEntityFilters = ({ showDates, datesSortedBy, displayDates }) => {
+	const { setShowDates, setDatesSortedBy, setDisplayDates } = {
+		setShowDates: () => null,
+		setDatesSortedBy: () => null,
+		setDisplayDates: () => null,
+	};
```

Comment:
These will come from the filter state manager, right?

Reply:
Absolutely, this is why I passed temp `() => null`